.github/actions/javascript/getArtifactInfo/getArtifactInfo.ts

```diff
@@ -1,13 +1,15 @@
-const _ = require('underscore');
-const core = require('@actions/core');
-const GithubUtils = require('../../../libs/GithubUtils');
+import core from '@actions/core';
```

Comment:
Same, I guess it should be [so](https://www.npmjs.com/package/@actions/core#import-the-package)
```suggestion
import * as core from '@actions/core';
```